app/javascript/components/pages/SponsorUs.jsx

```diff
@@ -1,47 +1,89 @@
 
 const SponsorUs = () => {
     return (
         <SharedLayout>
             <PageTitleWithContainer text="Sponsor Us" />
-            <div className="card-container flex flex-wrap justify-center">
-                {sponsorCardData.map((card) => {
+            <SponsorUsWidget />
+            <SomethingElseBanner />
+            <div className="w-full flex flex-row flex-wrap justify-center mt-32">
+                {infoCardData.map((card) => {
                     return (
+                        <SponsorUsInfoCard
+                            key={card.title}
+                            section={card.section}
+                            title={card.title}
+                        />
+                    );
+                })}
+            </div>
+            <SponsorsTile />
+        </SharedLayout>
+    );
+};
+
+const SponsorUsWidget = () => {
+    const [selectedLevel, setSelectedLevel] = useState(sponsorCardData[0].type);
+
+    return (
+        <div className="w-full flex flex-row justify-center mt-32 flex-wrap">
+            {sponsorCardData.map((card) => {
+                return (
+                    selectedLevel === card.type && (
                         <SponsorCard
                             key={card.type}
                             type={card.type}
                             returns={card.returns}
                             amount={card.amount}
-                        >
-                            <card.icon className="mx-auto p-2 h-20" />
-                        </SponsorCard>
-                    );
-                })}
-            </div>
-            <div className="flex flex-col items-center my-24">
-                {infoCardData.map((card) => {
+                            icon={card.icon}
+                        />
+                    )
+                );
+            })}
+
+            <div className="flex flex-col justify-between h-max mt-5 md:mt-0 md:w-auto md:ml-7">
+                {sponsorCardData.map((card) => {
                     return (
-                        <SponsorUsInfoCard
-                            key={card.title}
-                            section={card.section}
-                            title={card.title}
+                        <button
+                            key={`${card.type}-options`}
+                            onClick={() => setSelectedLevel(card.type)}
                         >
-                            <card.icon className="m-2" />
-                        </SponsorUsInfoCard>
+                            <SponsorshipOption
+                                title={card.type}
+                                amount={card.amount}
+                                selected={card.type === selectedLevel}
+                            />
+                        </button>
                     );
                 })}
             </div>
-            <SponsorCTA />
-        </SharedLayout>
+        </div>
+    );
+};
+
+const SomethingElseBanner = () => {
```

Comment:
I can see why you renamed this component but when I first saw it being called I thought it was placeholder text lolol. I noticed that it's pretty close to the `Sponsor Us Banner` from the job board, is there a reason it can't be a shared component?

Reply:
😁 you right! I'll make this change.